libs/SalesforceSDKCore/SalesforceSDKCore/Classes/URLHandlers/SFSDKAdvancedAuthURLHandler.m

```diff
@@ -40,13 +42,13 @@ - (BOOL)canHandleRequest:(NSURL *)url options:(NSDictionary *)options {
    
     // Not IDP enabled but Auth Code is present represents an Advanced Auth Flow situations. Ensure that we are not looking at any errors in either case.
     return (!preferences.idpEnabled &&
```

Comment:
`idpEnabled` is based on the IDP url scheme being set but that’s not used for the keychain based IDP -> SP flow. I did this because we know the format of the request for that case but we could also keep the `idpKeychainGroup` property or something similar